### PR TITLE
Cleaning

### PR DESCRIPTION
**Pull request by alxbilger**
- Remove the commented code
- Remove the manual timings

### CODE REVIEW

File: src/ModelOrderReduction/component/forcefield/HyperReducedTetrahedronHyperelasticityFEMForceField.inl
Comment (VannesteFelix) [inline src/ModelOrderReduction/component/forcefield/HyperReducedTetrahedronHyperelasticityFEMForceField.inl:1]: You remove the msg_info due to the bad timer but you didn't put another one to replace it.

**Comment by VannesteFelix** [comment]
Thanks for the cleaning @alxbilger !
Can you take a look @olivier-goury ? Are all the commented code useless ?

File: src/ModelOrderReduction/component/forcefield/HyperReducedTetrahedronHyperelasticityFEMForceField.inl
Comment (alxbilger) [reply src/ModelOrderReduction/component/forcefield/HyperReducedTetrahedronHyperelasticityFEMForceField.inl:1]: You mean another timer? Usually we don't put timers in such a function. In this case, `addDForce` is called many times per time steps.

**Comment by olivier-goury** [comment]
Thanks for the PR @alxbilger !
I can't believe there were still some lines of code with "OliviersMapping" !

**Pull request merged**